css-brochure-form/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+
+
+<body>
+  <h3>LEARN MORE</h3>
+
+  <form>
+    <label for="email">COMPANY EMAIL</label>
+    <input type="text" name="email" value="sally@company.co">
+
+    <label for="name">NAME</label>
+    <input type="text" name="name" value="Sally Smith">
+
+    <label for="location">WHERE ARE YOU HIRING?</label>
+    <input type="text" name="location" value="City, Country">
+
+    <label for="hiring">HOW MANY ENGINEERS ARE YOU HIRIGN IN THE NEXT 6 MONTHS?
+    </label>
+    <select name="engineers" id="hiring">
+      <option value="1-2">1-2 Engineers</option>
+    </select>
```

Comment:
These labels are not properly configured to pair with their form control. The form control elements need an `id` attribute with the same value as the `for` attribute on the label they are paired with.